Backend/User/src/main/java/user/repository/JWTokenRepository.java

```diff
@@ -39,6 +42,7 @@ void updateJWTLastLogin(
      * @param uuid the UUID of the user to log out
      * @return the number of rows affected by the update operation
      */
+    @Lock(LockModeType.PESSIMISTIC_WRITE)
     @Procedure(name = "UpdateLogout")
     Integer updateLogout(@Param("p_uuid") String uuid);
```

Comment:
can i ask, for the logout feature are we blacklisting JWTs until they expire right, cos i might want to implement a functionnality where i call the logout ( to invalidate ) if your service detects an unauthorised exception

Reply:
Yep store until expire